app/controllers/events_controller.rb

```diff
@@ -0,0 +1,49 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+    @user = current_user
+    @future_event = future_event?
+  end
+
+  def create
+    @event = Event.new(event_params)
```

Comment:
this is a requirement:
* [ ] uses #build method on association, i.e. current_user.events.build(event_params)
with that we have the user relationship already set.
- remember to use render when it fails not redirect